engine.py

```diff
@@ -180,6 +177,8 @@ def post_app_init(self):
         QtCore.QTextCodec.setCodecForCStrings(utf8)
         self.log_debug("set utf-8 codec for widget text")
 
+        # set the stylesheet
+        self._initialize_dark_look_and_feel()
```

Comment:
My question when reading this code is: *Why do we need to initialize the dark look and feel for houdini? normally we only do this for engines which haven't got a well defined style state set internally.* 

I think it would be cool to explain the strategy a little bit here. How houdini is theming its UI based on qss but this approach conflicts with our approach of having a two layers: underneath a dark QStyle and then on top of that qss modifications - this keeps the qss to a minimum and means that we only need to focus on defining the stuff that is different in qss - an app using an empty stylesheet will get a "neutral" dark appearance. 

I think it's also worth explaining that this has a global effect since we are setting a QStyle for the entire application but there currently don't seem to be any side effects (no pun intended).